gunicorn/reloader.py

```diff
@@ -89,7 +89,7 @@ def add_extra_file(self, filename):
             if dirname in self._dirs:
                 return
 
-            self._watcher.add_watch(dirname, mask=self.event_mask)
+            self._watcher.__load_trees([dirname])
```

Comment:
Why is a list `[dirname]` passed here but just `dirname` below?

Reply:
InotifyTrees expects an array of paths

Reply:
Then why isn't it an array below?